docs/apache-airflow/alembic_revisions.rst

```diff
@@ -0,0 +1,328 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I don't think this file was added to the TOC. Am I right?

Reply:
Hi @mik-laj , I have not checked if we can generate this table automatically. Since we display them in the logs , I think there could be a way to generate them. 

This is not added yet to the table of contents.

Reply:
I believe it is good to merge for RC1 as is @mik-laj -> we can automate it the first time we add a new migration. WDYT?

Reply:
We can merge this now, but I don't find it difficult to automate the generation of these tables.
Code that gets the migration list:
https://github.com/sqlalchemy/alembic/blob/master/alembic/command.py
Custom directives that render templates based on jinja:
https://github.com/apache/airflow/blob/master/docs/exts/operators_and_hooks_ref.py#L176-L205 + [`tabulate`](https://pypi.org/project/tabulate/) with rst output

Reply:
We need to have it tomorrow @ernest-kr -> do you think you can automate it ? BTW. You will also have to include https://github.com/apache/airflow/pull/12890  when merged

Reply:
Hi , Unfortunately I do not have time to finish by tomorrow on this. Can you create a new issue for the automate task so I can work on it?

Reply:
No problem. Will do :)